src/Umbraco.Community.Contentment/Umbraco.Community.Contentment.csproj

```diff
@@ -272,6 +272,7 @@
       <Private>False</Private>
     </Reference>
     <Reference Include="System.Xml" />
+    <Reference Include="System.Xml.Linq" />
```

Comment:
Is "System.Xml.Linq" reference required? From looking at the code changes, it doesn't appear to be needed.